.github/dependabot.yml

```diff
@@ -0,0 +1,133 @@
+version: 2
+updates:
+  # Go modules
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+    open-pull-requests-limit: 10
+    groups:
+      kubernetes:
+        patterns:
+          - "k8s.io/*"
+          - "sigs.k8s.io/*"
```

Comment:
@tenzen-y Do you know why do you add k8s packages to ignores in MPI operator for dependabot?
https://github.com/kubeflow/mpi-operator/blob/master/.github/dependabot.yml#L8

Reply:
It's probably because K8s updates are usually done via manual update.

For JobSet/Kueue we disable minor version bumps (1.28 to 1.29) but leave it for patches within a minor version.